src/clr_bundle_rm.c

```diff
@@ -89,7 +108,17 @@ enum swupd_code bundle_remove_main(int argc, char **argv)
 	}
 	progress_init_steps("bundle-remove", steps_in_bundleremove);
 
-	ret = remove_bundles(bundles);
+	remove_set_option_force(cmdline_option_force);
+
+	/* move the bundles provided in the command line into a
+	 * list so it is easier to handle them */
+	for (; *bundles; ++bundles) {
```

Comment:
Can building a new list just for providing as arg for remove_bundles be alright? Can this potentially be a bottle neck? I know we increase time_complexity only by o(n) but is it worth to avoid that?

Reply:
we need to manipulate that list in the bundle removal process, that is why I moved this to a list, I don't see how this could be a bottle neck. Maybe I'm not getting your question correctly.

Reply:
I mean we are re-creating a list by traversing a pointer to a char ..so it takes an additional space and requires 2 * n traversing of the items. But since you manipulate the list & require it, thats perfect

Reply:
yeah, the bundles provided by the user are not in a list, is just an array of chars with all bundles in the same array: "bundle1 bundle2 bundle3 etc", so I'm passing each one of those bundles into a list so we can manipulate them easier. The data in the list still points to each one of those elements in the original array. So basically I'm just adding a pointer to each one of those names inside the array.